fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -1299,42 +1305,9 @@ private boolean unprotectAbortTransaction(long transactionId, boolean abortPrepa
         transactionState.setReason(reason);
         transactionState.setTransactionStatus(TransactionStatus.ABORTED);
         unprotectUpsertTransactionState(transactionState, false);
-        for (PublishVersionTask task : transactionState.getPublishVersionTasks().values()) {
-            if (task != null) {
-                AgentTaskQueue.removeTask(task.getBackendId(), TTaskType.PUBLISH_VERSION, task.getSignature());
-            }
-        }
```

Comment:
Why remove?

Reply:
`transactionState.getPublishVersionTasks()` would always be empty here.

Reply:
how to guarantee always be empty?

Reply:
Only transactions of committed state will create publish version tasks but the transaction state must not be committed here, as you can see from the above lines of codes.